pages/common/yt-dlp.md

```diff
@@ -22,16 +22,16 @@
 
 - Specify audio format and audio quality of extracted audio (between 0 (best) and 10 (worst), default = 5):
 
-`yt-dlp --extract-audio --audio-format {{mp3}} --audio-quality {{0}} "{{https://www.youtube.com/watch?v=oHg5SJYRHA0}}"`
+`yt-dlp --extract-audio --audio-format {{mp3}} --audio-quality {{0}} "{{url}}"`
 
 - Download all playlists of YouTube channel/user keeping each playlist in separate directory:
 
-`yt-dlp -o "{{%(uploader)s/%(playlist)s/%(playlist_index)s - %(title)s.%(ext)s}}" "{{https://www.youtube.com/user/TheLinuxFoundation/playlists}}"`
+`yt-dlp -o "{{%(uploader)s/%(playlist)s/%(playlist_index)s - %(title)s.%(ext)s}}" "{{url}}"`
```

Comment:
this isn't the best of changes, how is the user to know that a playlist is needed

Reply:
I agree.

@thrila we try to be as clear as possible, and we use descriptive placeholder. I don't see any benefits on removing the example from the placeholder.

What about reverting this change?